sr_robot_commander/src/sr_robot_commander/sr_arm_commander.py

```diff
@@ -95,3 +97,23 @@ def _set_ground(self):
         pose.header.stamp = get_rostime()
         pose.header.frame_id = self._robot_commander.get_root_link()
         self._planning_scene.add_box("ground", pose, (3, 3, 0.1))
+
+    def get_pose_reference_frame(self):
+        return self._move_group_commander.get_pose_reference_frame()
+
+    def plan_cartesian_path_to_pose(self, target_pose, min_fraction=1, eef_step=.01, jump_threshold=1000):
+        if type(target_pose) != Pose:
+            rospy.logerr("Target should be Pose.")
+            return None
+
+        start_pose = self._move_group_commander.get_current_pose().pose
+
+        (self._SrRobotCommander__plan, fraction) = self._move_group_commander.compute_cartesian_path(
+            [start_pose, target_pose], eef_step, jump_threshold)
+
+        if fraction < min_fraction:
+            rospy.logerr("Couldn't reach enough waypoints, only %f" % fraction)
+            self._SrRobotCommande__path = None
```

Comment:
@dg-shadow it would be nice to have such protected function. Even something like _clean_plan(). Name mangling algorithm might change in the future versions of Python and your code will not work :)